### ISSUE TITLE
Display human-readable dates on contributing page

### ISSUE THREAD

**Issue by ErikSchierboom** [exercism/v3-dashboard#48]
The contributing page currently displays the last updated dates as the UTC-formatted string that is returned by the GitHub API. It would be nice if the dates were formatted to be more human-readable.

**Comment by JesseSingleton** [comment]
@ErikSchierboom I'll take this too. Do we have an idea of what format we want?

Without going too deeply into the rabbit hole of Timezones, my thought was we wrap the returned date in the JS `Date` object and use one of these two methods depending on if we want date and time or just date.

```
new Date('2020-04-23T20:25:26Z').toString()
"Thu Apr 23 2020 21:25:26 GMT+0100 (British Summer Time)"
new Date('2020-04-23T20:25:26Z').toDateString()
"Thu Apr 23 2020"
```

Thoughts?

P.S Slight bump for my PR #54 😄

**Comment by ErikSchierboom** [comment]
I think I prefer the latter, without any timezone issues. I'll assign the issue to you.

**Comment by ErikSchierboom** [comment]
Closed by #65
